src/test/java/com/epam/tc/hw2/ExerciseTwoTest.java

```diff
@@ -0,0 +1,78 @@
+package com.epam.tc.hw2;
+
+//import static com.epam.tc.hw2.ExerciseOneTest.*;
+import java.time.Duration;
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+
+
+public class ExerciseTwoTest extends AbstractTest {
+    private static String ID_DROPDOWN_SERVICE_MENU = "dropdown";
+    private static String XPATH_MENU_ITEM_DIFFERENT_ELEMENT = "//a[text()='Different elements']";
+    private static String XPATH_CHECKBOX_WATER = "//label[text()[contains(., ' Water')]]/*[@type='checkbox']";
+    private static String XPATH_CHECKBOX_WIND = "//label[text()[contains(., ' Wind')]]/*[@type='checkbox']";
+    private static String XPATH_RADIOBUTTON_SILVER = "//label[text()[contains(., ' Silver')]]/*[@type='radio']";
+    private static String XPATH_DROPDOWN_BLUE = "//*[text()='Blue']";
+    private static String CSS_LOGS = ".panel-body-list > li";
+
```

Comment:
`.panel-body-list.logs  > li` This is how you can choose several classes using css